modules/alloy/src/main/ts/ephox/alloy/behaviour/docking/Dockables.ts

```diff
@@ -54,40 +54,30 @@ const disappear = (component: AlloyComponent, contextualInfo: DockingContext): v
   contextualInfo.onHide(component);
 };
 
-const isPartiallyVisible = (box: Boxes.Bounds, viewport: Boxes.Bounds): boolean => {
-  return box.y < viewport.bottom && box.bottom > viewport.y;
-};
+const isPartiallyVisible = (box: Boxes.Bounds, viewport: Boxes.Bounds): boolean => box.y < viewport.bottom && box.bottom > viewport.y;
 
-const isTopCompletelyVisible = (box: Boxes.Bounds, viewport: Boxes.Bounds): boolean => {
-  return box.y >= viewport.y;
-};
+const isTopCompletelyVisible = (box: Boxes.Bounds, viewport: Boxes.Bounds): boolean => box.y >= viewport.y;
 
-const isBottomCompletelyVisible = (box: Boxes.Bounds, viewport: Boxes.Bounds): boolean => {
-  return box.bottom <= viewport.bottom;
-};
-
-const isVisibleForModes = (modes: DockingMode[], box: Boxes.Bounds, viewport: Boxes.Bounds): boolean => {
-  return Arr.forall(modes, (mode) => {
-    switch (mode) {
-      case 'bottom':
-        return isBottomCompletelyVisible(box, viewport);
-      case 'top':
-        return isTopCompletelyVisible(box, viewport);
-    }
-  });
-};
+const isBottomCompletelyVisible = (box: Boxes.Bounds, viewport: Boxes.Bounds): boolean => box.bottom <= viewport.bottom;
 
-const getPrior = (elem: Element<HTMLElement>, state: DockingState): Option<Boxes.Bounds> => {
-  return state.getInitialPosition().map((pos) => {
-    // Only supports position absolute.
-    return Boxes.bounds(
-      pos.bounds.x,
-      pos.bounds.y,
-      Width.get(elem),
-      Height.get(elem)
-    );
-  });
-};
+const isVisibleForModes = (modes: DockingMode[], box: Boxes.Bounds, viewport: Boxes.Bounds): boolean => Arr.forall(modes, (mode) => {
+  switch (mode) {
+    case 'bottom':
+      return isBottomCompletelyVisible(box, viewport);
+    case 'top':
+      return isTopCompletelyVisible(box, viewport);
+  }
+});
+
+const getPrior = (elem: Element<HTMLElement>, state: DockingState): Option<Boxes.Bounds> => state.getInitialPosition().map((pos) =>
+// Only supports position absolute.
```

Comment:
The indentation for the comment here is now incorrect.

Reply:
Fixed